### PR TITLE
Fix/gallery modal navigation inconsistencies

### PR DESCRIPTION
**Pull request by thorbrink**
* Fix: Gallery instance being reused on every modal open and causing gallery steps index inconcistenties.
* Fix: Multiple Gallery components being used as 1 when opening modal.
* Add: Testing framework for testing JS.
* Add: Tests for gallery.js
* Add: README.md

### CODE REVIEW

**Comment by helsingborgsstad** [comment]
Task linked: [CU-3xxv08c Bugg i galleri (flertal)](https://app.clickup.com/t/3xxv08c)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+## Summary
+The style guide is intended for websites within Helsingborgs stad and others who use our platform. The guide provides examples, markup and themes for our standardized components. The Helsingborg Styleguide is a flexible and minimalistic component-based framework built in the BEM standard & designed around the Atomic Design principle.
+
+### Requirements
+
+- NodeJS >= 16
+- PHP >= 7.4
+- [Composer](https://getcomposer.org/)
+
+## Installation
```
Comment (thorbrink) [inline README.md:31]: @sebastianthulin could you please add information to "Installation" to make it complete?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+1. Clone the repo:
+  ```sh
+  git clone git@github.com:helsingborg-stad/styleguide.git
+  ```
+2. Run the build script in the cloned project:
+  ```sh
+  cd styleguide && php build.php
+  ```
+
+## Development
```
Comment (thorbrink) [inline README.md:41]: @sebastianthulin could you please add information to "Development" to make it complete?

File: README.md (hunk 1 above)
Comment (yo-l1982) [reply README.md:31]: Added a little fix for the script installing the component library.
Installation instructions is comlete imo.

File: README.md (hunk 2 above)
Comment (yo-l1982) [reply README.md:41]: We could add a something about getting it running on a web server.
I just use the `php -S localhost:8000` in the root to get this up for dev.

**Review by yo-l1982 — APPROVED** [review]
Docs and setup instructions is goodie

**Pull request merged**